app/views/home/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="home-buttons"><%= link_to markets_path, class: 'btn btn-success btn-lg' do %>
+Market
+<% end %>
+
+<%= link_to vendors_path, class: 'btn btn-success btn-lg' do %>
+Vendor
+<% end %>
+
+<div class="welcome"></div>
+</div>
```

Comment:
Thinking this might be the bootstrap/heroku issue. I tried to fix it a bit really quickly, but alas, damn that photo.